tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -24,3 +33,23 @@ Verify User Can Spawn A Notebook
     Wait For RHODS Dashboard To Load
     Launch JupyterHub Spawner From Dashboard
     Spawn Notebook With Arguments    image=s2i-minimal-notebook    size=Default
+
+Verify User Is Unable To Spawn Notebook
+    [Documentation]    Verifies User is unable to access notebooks in jupyterhub
+    Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+    Login To RHODS Dashboard    ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    ${TEST_USER.AUTH_TYPE}
+    Wait For RHODS Dashboard To Load
+    Menu.Navigate To Page    Applications    Enabled
+    Launch JupyterHub From RHODS Dashboard Link
+    Login To Jupyterhub  ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    ${TEST_USER.AUTH_TYPE}
+    ${authorization_required} =  Is Service Account Authorization Required
+    Run Keyword If  ${authorization_required}  Authorize jupyterhub service account
+    Fix Spawner Status
+    Wait Until Page Contains Element  xpath://span[@id='jupyterhub-logo']
+    Run Keyword And Expect Error    *    Wait Until JupyterHub Spawner Is Ready
```

Comment:
still not sure about this. It should be complemented by a check on the error the user is expected to see in the page.

Reply:
Updated